pysat/tests/test_orbits.py

```diff
@@ -753,8 +650,41 @@ def teardown(self):
         """Runs after every method to clean up previous testing."""
         del self.testInst, self.stime
 
+    def test_repeat_orbit_calls_asym_multi_day_0_UT_really_long_time_gap(self):
+        """Test successful orbit calls over a series of lengthening gaps
+        """
+        self.testInst.load(date=self.stime)
```

Comment:
The loaded data and the number of orbits determines which gap we run into. Based on my list of times, we should load this one on the 19th and go for like 45 orbits.

Reply:
@rstoneback, I've added the check for where the orbit breaks are here, and it is failing for the 45 orbit test on the 19th.  We could drop these added tests for now and add an issue.

Reply:
UPDATE: Since this is a new test, I've dropped the check and added #861 to document